src/PortfolioPlanning/Common/Services/BacklogConfigurationDataService.ts

```diff
@@ -46,7 +52,8 @@ export class BacklogConfigurationDataService {
             effortFieldRefName: projectEfforFieldRefName,
 
             orderedWorkItemTypes: portfolioLevelsData.orderedWorkItemTypes,
-            backlogLevelNamesByWorkItemType: portfolioLevelsData.backlogLevelNamesByWorkItemType
+            backlogLevelNamesByWorkItemType: portfolioLevelsData.backlogLevelNamesByWorkItemType,
+            workItemTypeMappedStates: workItemTypeMappedStates
```

Comment:
Another thing to consider is that you are retrieving all state categories for all work item types, when you really only need the names of the `InProgress` states in the `DependencyPanel`... maybe just return a map of work item types and the names of the `InProgress` states will suffice?